node/localnode.go

```diff
@@ -188,6 +168,31 @@ func NewLocalNode(wallet vault.Wallet, nn *nnet.NNet) (*LocalNode, error) {
 	return localNode, nil
 }
 
+func (localNode *LocalNode) MarshalJSON() ([]byte, error) {
```

Comment:
Why moving this function from L51, any difference?

Reply:
It's the same, but just a convention to put receiver function after create function (NewXXX)